perception/tensorrt_yolox/src/tensorrt_yolox.cpp

```diff
@@ -71,20 +68,22 @@ TrtYoloX::TrtYoloX(
     std::accumulate(input_dims.d + 1, input_dims.d + input_dims.nbDims, 1, std::multiplies<int>());
   if (needs_output_decode_) {
     const auto output_dims = trt_common_->getBindingDimensions(1);
-    input_d_ = cuda_utils::make_unique<float[]>(batch_config[2] * input_size);
+    input_d_ = cuda::memory::device::make_unique<float[]>(batch_config[2] * input_size);
     out_elem_num_ = std::accumulate(
       output_dims.d + 1, output_dims.d + output_dims.nbDims, 1, std::multiplies<int>());
     out_elem_num_per_batch_ = static_cast<int>(out_elem_num_ / batch_config[2]);
-    out_prob_d_ = cuda_utils::make_unique<float[]>(out_elem_num_);
-    out_prob_h_ = cuda_utils::make_unique_host<float[]>(out_elem_num_, cudaHostAllocPortable);
+    out_prob_d_ = cuda::memory::device::make_unique<float[]>(out_elem_num_);
+    out_prob_h_ = cuda::memory::host::make_unique<float[]>(out_elem_num_);
```

Comment:
```suggestion
    out_prob_h_ = cuda::memory::host::make_unique<float[]>(out_elem_num_);
    cuda::memory::host::register_(out_prob_h_.get(), out_prob_h_.size(), CU_MEMHOSTREGISTER_PORTABLE);
```
I guess this `register_` call might be needed to make an equivalent code (?) (Sorry for my unfamiliarity with the usage of this wrapper). I'd appreciate your consideration.